app/views/assignments/_submission_enabled_toggles.html.erb

```diff
@@ -0,0 +1,20 @@
+<% toggles = gradesheet.assignment.submission_enabled_toggles %>
```

Comment:
This seems very indirect.  Couldn't you use `@assignment` and `@course` here directly?  Why go through the gradesheet?